src/bindings/js/node/src/model_wrap.cpp

```diff
@@ -128,3 +129,13 @@ Napi::Value ModelWrap::is_dynamic(const Napi::CallbackInfo& info) {
     const auto result = _model->is_dynamic();
     return Napi::Boolean::New(env, result);
 }
+
+Napi::Value ModelWrap::get_output_size(const Napi::CallbackInfo& info) {
+    Napi::Env env = info.Env();
+    if (info.Length() > 0) {
+        reportError(env, "getOutputSize() does not accept any arguments.");
+        return env.Null();
```

Comment:
Better to use undefined, because `typeof null == 'object'`, but in the case of error it's really `undefined`
```suggestion
        return env.Undefined();
```